app/Models/EntryDAO.php

```diff
@@ -800,6 +781,75 @@ protected function sqlListEntriesWhere($alias = '', $filters = null, $state = Fr
 					$sub_search .= ') ';
 				}
 
+				if ($filter->getFeedIds()) {
```

Comment:
Updated to be exact same model as label IDs although it does not make a logical difference for feed IDs because an article cannot be in more than one feed